scipy/integrate/_quadrature.py

```diff
@@ -1035,3 +1038,131 @@ def newton_cotes(rn, equal=0):
+                   "integration limits.")
+        raise ValueError(message) from e
+
+    try:
+        func(*ranges, *args)
+        vfunc = func
+    except Exception as e:
+        wmessage = ("Exception encountered when attempting vectorized call to "
+                    f"`func`: {e}. `func` should accept arrays `x0, ..., xn` "
+                    "for better performance.")
+        emessage = ("Exception encountered when attempting vectorized call to "
+                    f"`func`: {e}, and `args` is not `None`. `func` must "
+                    "accept arrays `x0, ..., xn` if `args` is not `None`.")
+        if args != tuple():
+            raise ValueError(emessage)
+        else:
+            warnings.warn(wmessage)
+        vfunc = np.vectorize(func)
+
+    n_points_int = np.int64(n_points)
+    if n_points != n_points_int:
+        message = "`n_points` must be an integer."
+        raise TypeError(message)
+
+    if qmc_engine is None:
+        qmc_engine = stats.qmc.Halton(dim)
+    elif not isinstance(qmc_engine, stats.qmc.QMCEngine):
+        message = "`qrng` must be an instance of scipy.stats.qmc.QMCEngine."
+        raise TypeError(message)
+
+    return vfunc, ranges, n_points_int, qmc_engine, args, stats
+
+
+QMCQuadResult = namedtuple('QMCQuadResult', ['integral', 'standard_error'])
+
+
+def qmc_quad(func, ranges, *, n_points=1024, qrng=None, args=None):
+    """
+    Compute an integral in N-dimensions using Quasi-Monte Carlo quadrature.
+
+    Parameters
+    ----------
+    func : callable
+        The function to be integrated. Must accept separate arguments
+        ``x0, ..., xn`` corresponding with the coordinate in each dimension.
+        May also accept additional positional arguments specified with
+        `args`.
+    ranges : array-like
+        A two-dimensional array specifying the lower and upper integration
+        limits of each variable.
+    n_points : int, default: 1024
+        The number of Quasi-Monte Carlo points at which to evaluate the
+        integrand.
+    qrng : stats.qmc.QMCEngine, optional
+        An instance of the QMCEngine from which to sample QMC points.
+        The QMCEngine must be initialized to a number of dimensions
+        corresponding with the number of variables ``x0, ..., xn`` passed to
+        `func`. If a QMCEngine is not provided, `scipy.stats.qmc.Halton` will
+        be initialized with the number of dimensions determine from `ranges`
+        and the default `seed`.
+    args : sequence, optional
+        Extra arguments to pass to `func`. All extra arguments must be packed
+        into a sequence; the will be unpacked to be passed to `func`, e.g.
+        ``func(x1, x2, *args)``.
+
+    Returns
+    -------
+    result : object
+        The a result object with attributes:
+
+        integral : float
+            The estimate of the integral
+        standard_error :
+            The standard error of the
```

Comment:
```suggestion
            The standard error of the mean of the integrand evaluated at the QMC-sampled quadrature points.
```
scaled by the area of the integration region?

I added this at the last minute for a rough error tolerance. Perhaps QMC theory can give us a better bound? But I know it depends on the properties of the integrand.